### PR TITLE
Styles don’t load after update from 1.6.7 to 1.6.8/1.6.9

### PR DESCRIPTION
**Pull request by gvgvgvijayan**
#403 Fixed the issue styles not applied for view page of the elementor page

### CODE REVIEW

File: inc/elementor/kit/Manager.php [hunk 1]
```diff
@@ -116,7 +117,16 @@ public function is_using_custom_kit() {
 		}
 
 		// Return early if Global kit and current kit is same.
-		if ( (int) Options::get_instance()->get( 'global_kit' ) === (int) $kit_id ) {
+		$global_kit = Options::get_instance()->get( 'global_kit' );
+		if ( ! $global_kit ) {
+
+			$el_manager = new EL_Manager();
+			$global_kit = $el_manager->get_active_id();
```
Comment (mauryaratan) [inline inc/elementor/kit/Manager.php:124]: Could rewrite as following?

```suggestion
			$global_kit = ( new EL_Manager() )->get_active_id();
```

File: inc/elementor/kit/Manager.php [hunk 2]
```diff
@@ -12,6 +12,7 @@
 use Analog\Plugin;
 use Analog\Utils;
 use Elementor\Core\Files\CSS\Post as Post_CSS;
+use Elementor\Core\Kits\Manager as EL_Manager;
```
Comment (mauryaratan) [inline inc/elementor/kit/Manager.php:15]: `KitManager` would make more sense here

**Comment by gvgvgvijayan** [comment]
> Please take care of comments

Ok, I will update that.

**Comment by mauryaratan** [comment]
💯

**Pull request merged**